content/how-to/bpmn/debug-workflows.md

```diff
@@ -0,0 +1,249 @@
+---
+title: Handle errors and debug
+date: '2024-04-24T19:35:09+03:00'
+categories: ["how-to"]
+description: Strategies to handle errors in your BPMN workflows, and ways to debug workflows when things don't work as expected.
+weight: 350
+menu:
+  main:
+    parent: howto-bpmn
+    identifier:
+---
+
+Errors come in two categories: expected and unexpected.
+The Rhize BPMN engine has ways to handle both.
+
+A robust workflow is going to have built-in logic to anticipate errors.
+For unexpected issues, every node in a workflow is _instrumented_, meaning it emits  traces about its performance.
```

Comment:
```suggestion
For unexpected issues, every node in a workflow is _instrumented_, meaning it emits traces so that you can observe its behavior and performance.
```